runtime/zeitgeist/src/integration_tests/xcm/tests/transfers.rs

```diff
@@ -405,200 +440,147 @@ fn transfer_eth_zeitgeist_to_sibling() {
         ));
 
         // Confirm that Alice's balance is initial_balance - amount_transferred
-        assert_eq!(AssetManager::free_balance(ETH_ID.into(), &ALICE), 0);
+        let alice_balance = AssetManager::free_balance(ETH_ID.into(), &alice());
+        let alice_expected = alice_initial_balance - adjusted_balance(eth(1), transfer_amount);
+        assert_eq!(alice_balance, alice_expected);
     });
 
     Sibling::execute_with(|| {
-        let fee_adjusted = adjusted_balance(eth(1), eth_fee());
-        let expected = transfer_amount - fee_adjusted;
+        let expected = bob_initial_balance + transfer_amount - adjusted_balance(eth(1), eth_fee());
+        let expected_sovereign = initial_sovereign_balance - transfer_amount;
 
         // Verify that Bob now has initial balance + amount transferred - fee
-        assert_eq!(Balances::free_balance(&BOB), sibling_bob_initial_balance + expected,);
-
+        assert_eq!(Balances::free_balance(&bob()), expected);
         // Verify that the amount transferred is now subtracted from the zeitgeist account at sibling
```

Comment:
I think we're missing the treasury balance check here.

Reply:
I think it's not needed here because from the perspective of `Sibling` it is just receiving it's native token back (ZTG).